packages/api/src/external/cda/generate-ccd.ts

```diff
@@ -1,16 +1,30 @@
 import { Bundle } from "@medplum/fhirtypes";
+import { FHIR_BUNDLE_SUFFIX, createUploadFilePath } from "@metriport/core/domain/document/upload";
 import { Patient } from "@metriport/core/domain/patient";
+import { S3Utils } from "@metriport/core/external/aws/s3";
 import { metriportDataSourceExtension } from "@metriport/core/external/fhir/shared/extensions/metriport";
 import { getOrganizationOrFail } from "../../command/medical/organization/get-organization";
 import { getConsolidatedPatientData } from "../../command/medical/patient/consolidated-get";
 import { convertFhirToCda } from "../../command/medical/patient/convert-fhir-to-cda";
 import { bundleSchema } from "../../routes/medical/schemas/fhir";
+import { Config } from "../../shared/config";
 import { toFHIR as toFhirOrganization } from "../fhir/organization";
 import { validateFhirEntries } from "../fhir/shared/json-validator";
 import { generateEmptyCcd } from "./generate-empty-ccd";
 
+const region = Config.getAWSRegion();
+const bucket = Config.getMedicalDocumentsBucketName();
+const s3Utils = new S3Utils(region);
+
 export async function generateCcd(patient: Patient): Promise<string> {
-  // TODO: 1905 - check on s3 whether FHIR_BUNDLE.json files exist
+  const uploadsExist = await s3Utils.fileExists(bucket, {
+    targetString: FHIR_BUNDLE_SUFFIX,
+    path: createUploadFilePath(patient.cxId, patient.id, ""),
+  });
+  if (!uploadsExist) {
+    return generateEmptyCcd(patient);
+  }
```

Comment:
can you explain this logic here a little more? Why are we checking for FHIR bundles on s3? Why are we storing FHIR bundles on s3?

Reply:
We're storing FHIR bundles on S3 to have a paper trail for where CX-contributed data come from. So, every time a CX contributes data, we are storing it on S3. 
Additionally, this provides us with a quick mechanism to see if a patient has CX-contributed data - if there are `FHIR_BUNDLE` files on S3, that means we should make a real CCD. Otherwise, we can make an empty one.